api/libretime_api/settings/prod.py

```diff
@@ -73,3 +73,20 @@
 TIME_ZONE = "UTC"
 USE_I18N = True
 USE_TZ = True
+
+# Email
+# https://docs.djangoproject.com/en/4.2/topics/email/
+
+EMAIL_BACKEND = "django.core.mail.backends.smtp.EmailBackend"
+
+EMAIL_HOST = CONFIG.email.host
+EMAIL_PORT = CONFIG.email.port
+EMAIL_HOST_USER = CONFIG.email.user
+EMAIL_HOST_PASSWORD = CONFIG.email.password
+EMAIL_USE_TLS = CONFIG.email.security == "explicit"
+EMAIL_USE_SSL = CONFIG.email.security == "implicit"
```

Comment:
`use_tls` or `use_ssl` are somewhat difficult to understand, so I changed this to use one of `None|implicit|explicit`.

- None is no tls encryption
- implicit is tls encryption
- explicit is tls encryption using starttls

Reply:
Hmm. I would maybe change the options to `None|SSL/TLS|STARTTLS` - those seem to be more consistent other email setting implementations (for example email clients, Nextcloud)

Reply:
Yes, the options you proposed are better! I changed it.